metadata-ingestion/src/datahub/ingestion/glossary/datahub_classifier.py

```diff
@@ -73,6 +73,14 @@ class Config:
         description="Factors and their weights to consider when predicting info types",
         alias="prediction_factors_and_weights",
     )
+    StripExclusionFormatting: bool = Field(
+        default=True, alias="strip_exclusion_formatting"
+    )
```

Comment:
`StripExclusionFormatting` is not an infotype level config. As per equivalent datahub-classify PR, its at global level and is set only once across all infotypes. Did you intend this global behavior ? In that case ->  `strip_exclusion_formatting` should be in `DataHubClassifierConfig` class in same file . Also, need not use `StripExclusionFormatting` with alias `strip_exclusion_formatting` here . You can directly name the field as `strip_exclusion_formatting` .

Reply:
done!